src/PacketHandler/PacketEntities.ts

```diff
@@ -94,7 +94,7 @@ function handleEntity(entity: PacketEntity, match: Match) {
 					}
 				}
 				if (prop.definition.ownerTableName === 'm_iAmmo') {
-					if (prop.value > 0) {
+					if (prop.value && prop.value > 0) {
```

Comment:
Made it more specific. The type issue is that prop.value can be null there, so checking `prop.value > 0` is throwing `error TS2531: Object is possibly 'null'`